docs/api/dashboard-new.md

```diff
@@ -10,7 +10,7 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
```

Comment:
Do you need to remove "(New)" from this article title and sidebar label?

Reply:
Yes John. We need to just keep it as Dashboard instead of Dashboard New.